addons/binding/org.openhab.binding.velux/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	* Copyright 2016-2018 (c) Guenther Schreiner <guenther.schreiner@smile.de>
+	*
+	* All rights reserved. This program and the accompanying materials
+	* are made available under the terms of the Eclipse Public License v1.0
+	* which accompanies this distribution, and is available at
+	* http://www.eclipse.org/legal/epl-v10.html
+	*
+-->
+<!-- Velux Bridge Things Binding -->
+<thing:thing-descriptions bindingId="velux" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Velux Bridge Binding -->
+	<bridge-type id="klf200">
+		<label>Velux KLF200 Bridge</label>
+		<description>The Velux KLF200 represents a gateway to all Velux devices.</description>
+
+		<channels>
+			<channel id="status" typeId="status" />
+			<channel id="firmware" typeId="firmware" />
+			<channel id="ipAddress" typeId="ipAddress" />
+			<channel id="subnetMask" typeId="subnetMask" />
+			<channel id="defaultGW" typeId="defaultGW" />
+			<channel id="DHCP" typeId="DHCP" />
+			<channel id="WLANSSID" typeId="WLANSSID" />
+			<channel id="WLANPassword" typeId="WLANPassword" />
+			<channel id="scenes" typeId="scenes" />
+			<channel id="products" typeId="products" />
+			<channel id="doDetection" typeId="doDetection" />
```

Comment:
what is this channel for? Does an enduser trigger this during normal operation? I would expect this to be a job for the discovery service.

Reply:
Could be eventually eliminated - leftover from openHAB1 binding where is explicit possibility is needed to let the user start the discovery.